update deps for wallet vc lips

Ogenbertrand opened the pull request:
I created a Script to automate the process of updating the dependencies for the wallet-vc-libs, i also restructured the eslint conffig files for all the packages to suite the newest version.
Please go through it and live ya thoughts.

I addressed the comments forkim left, but due to a lot of conflicts i could not resolve from the package-lock, i had to create another branch for this implementation. I'll like the reviews to please take their time in reviewing this PR since there are a lot of updates on  it. al so please test each component to ensure everything works fine.

Ogenbertrand commented:
@hugoib, @IngridPuppet , @forkimenjeckayang , and @Awambeng please review this PR.

IngridPuppet commented on eslint.config.mjs:
```diff
@@ -0,0 +1,55 @@
+  {
+    ignores: ['!**/*', '**/node_modules/', '**/dist'],
+  },
+  ...compat.extends(
+    'eslint:recommended',
+    'plugin:@typescript-eslint/recommended',
+  ),
```
Please make sure you go through the following documentation on new `eslint` flat configuration format to better address my point here: https://typescript-eslint.io/packages/typescript-eslint#config

I'm certain that these configs can be made significantly simplified. Take for example:
```js
import eslint from '@eslint/js';
import jsoneslint from 'eslint-plugin-jsonc';
import tseslint from 'typescript-eslint';

export default tseslint.config(
  eslint.configs.recommended,
  tseslint.configs.recommended,
  jsoneslint.configs['flat/recommended-with-json'],
  {
    ignores: ['dist'],
  },
);
```

IngridPuppet commented on package.json:
The main point of this ticket was actually to resolve the numerous warnings raised when running `npm install` on the project. Don't you think that can be addressed here?
![image](https://github.com/user-attachments/assets/30da4407-bca8-42d7-a38d-fd32d0e3fb4c)

Ogenbertrand commented:
I addressed the comments left for this PR, most importantly, deleted the script, and use the command "npx npm-check-updates --deep -u" to update the deps, all the deps are now updated except the react and react-dom, due to some compatebilities btw the usePWA and qr-scanner packages, i didn't update those pacakges, and moreover their latest version seems to be unstable. please review and live your thoughts.

Thank you.

IngridPuppet reviewed (changes requested):
There are still some points left unaddressed. Please mind leaving a comment so I can get your point.
